drivers/storage/portworx/component/autopilot.go

```diff
@@ -746,6 +750,10 @@ func (c *autopilot) getDesiredVolumesAndMounts(
 }
 
 func (c *autopilot) isAutopilotSecretCreated(namespace string) bool {
+	if c.isSecretCreated != nil {
```

Comment:
I don't think we need this flag at all because we anyway read the secret from the cache when we execute the `k8sClient.Get`.